src/qt/forms/debugwindow.ui

```diff
@@ -887,85 +890,112 @@
             <height>0</height>
            </size>
           </property>
-          <layout class="QVBoxLayout" name="verticalLayout_7">
-           <item>
-            <widget class="QTableView" name="peerWidget">
-             <property name="tabKeyNavigation">
-              <bool>false</bool>
-             </property>
-             <property name="alternatingRowColors">
-              <bool>true</bool>
-             </property>
-             <property name="textElideMode">
-              <enum>Qt::ElideMiddle</enum>
-             </property>
-             <property name="sortingEnabled">
-              <bool>true</bool>
-             </property>
-             <property name="wordWrap">
-              <bool>false</bool>
-             </property>
-             <attribute name="horizontalHeaderHighlightSections">
-              <bool>false</bool>
-             </attribute>
-            </widget>
-           </item>
+          <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
-            <widget class="QLabel" name="banHeading">
-             <property name="sizePolicy">
-              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
-               <horstretch>0</horstretch>
-               <verstretch>0</verstretch>
-              </sizepolicy>
-             </property>
-             <property name="minimumSize">
-              <size>
-               <width>0</width>
-               <height>32</height>
-              </size>
-             </property>
-             <property name="maximumSize">
-              <size>
-               <width>16777215</width>
-               <height>32</height>
-              </size>
-             </property>
-             <property name="font">
-              <font>
-               <pointsize>12</pointsize>
-              </font>
-             </property>
-             <property name="cursor">
-              <cursorShape>IBeamCursor</cursorShape>
-             </property>
-             <property name="text">
-              <string>Banned peers</string>
-             </property>
-             <property name="alignment">
-              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
-             </property>
-             <property name="wordWrap">
-              <bool>true</bool>
-             </property>
-             <property name="textInteractionFlags">
-              <set>Qt::NoTextInteraction</set>
+            <widget class="QSplitter" name="tableSplitter">
+             <property name="styleSheet">
+              <string notr="true">QSplitter::handle{background: #a6a6a6;}</string>
```

Comment:
Why this style?

Reply:
I did this to make the splitter visible and looks aesthetically pleasing.
Do you recommend keeping the splitter invisible?

Reply:
What do you think about this? @promag

Reply:
> Do you recommend keeping the splitter invisible?

I don't think the hardcoded color will work for any platform and appearance (dark/light), no?

Reply:
> I don't think the hardcoded color will work for any platform and appearance (dark/light), no?

The spiller looks fine for the default light mode (in Linux) and dark mode in macOS, which can be seen in the screenshots of the following comments: [light](https://github.com/bitcoin-core/gui/pull/457#issue-1035432748); [dark](https://github.com/bitcoin-core/gui/pull/457#pullrequestreview-791298779).

However, this doesn't guarantee that this will work for any custom appearance. Do you recommend removing the hardcoded color and making the splitter invisible again?

Reply:
Why not keeping the same look as for horizontal `splitter`?

> Do you recommend keeping the splitter invisible?

What does make you think so?

Reply:
> Do you recommend keeping the splitter invisible?

By this, I meant to say, keeping it the same as the horizontal splitter.

> Why not keep the same look as for horizontal splitter?

I think this will be a more optimal approach to go with.

Reply:
But the horizontal splitter is not invisible :)

And having different looks for similar GUI elements does not as an optimal design for me.

Reply:
At least, introducing a new GUI element, and changing thuis GUI element style across the GUI are two different tasks.

Reply:
Fixed splitter color to default palette in 0c364201a2e16355cd6424c6137761c7967db4e0